media/videostitcher/create_live_config_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package videostitcher
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	stitcherstreampb "cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+)
+
+func setupTestCreateLiveConfig(slateID string, t *testing.T) {
```

Comment:
Having the separate setup and cleanup methods for each test is nice but there seems to be a lot of duplicated code between these methods. Is it possible to factor out shared functions that do basic setup/cleanup of resources?

Reply:
Moved common create/delete test setup code to stitcher_test.go